### PR TITLE
Add Solution for Red Canary

### PR DESCRIPTION
**Pull request by rc-iwoodley**
**Description for the PR:**

[Red Canary website](https://redcanary.com).

Change(s):
- Added a Red Canary data connector, logo, and sample data.

**Testing Completed:**
Yes/ No : **Yes**

### CODE REVIEW

**Comment by v-sabiraj** [comment]
Hey @aprakash13 ,please review Analytic Rules, thanks.

**Comment by samstraka-redcanary** [comment]
Hey @aprakash13 , please let us know if you have any questions about the analytic, we are really excited to get this out!

**Review by aprakash13 — COMMENTED** [review]
Looks good from a query perspective. Please feel free to approve if other content looks good.

**Comment by v-sabiraj** [comment]
Hey @samstraka-redcanary , please check and confirm for the above changes, otherwise we are good to merge.

File: Solutions/RedCanary/Analytic Rules/RedCanaryThreatDetection.yaml [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  - DefenseEvasion
+  - Discovery
+  - Execution
+  - Exfiltration
+  - Impact
+  - InitialAccess
+  - LateralMovement
+  - Persistence
+  - PrivilegeEscalation
+  - PreAttack
```
Comment (aprakash13) [inline Solutions/RedCanary/Analytic Rules/RedCanaryThreatDetection.yaml:27]: PreAttack is not currently a supported tactics for the Sentinel queries. Here is a list that we currently support: https://attack.mitre.org/tactics/enterprise/ 

Please remove this from the tactics list.

**Review by aprakash13 — CHANGES_REQUESTED** [review]
Please make the changes in the Tactics section.

**Review by v-sabiraj — APPROVED** [review]

**Pull request merged**